dask/array/tests/test_fft.py

```diff
@@ -27,9 +31,21 @@ def key(k):
     "ihfft",
 ]
 
+all_nd_funcnames = [
+    "fft2",
+    "ifft2",
+    "fftn",
+    "ifftn",
+    "rfft2",
+    "irfft2",
+    "rfftn",
+    "irfftn",
+]
+
 nparr = np.arange(100).reshape(10, 10)
 darr = da.from_array(nparr, chunks=(1, 10))
 darr2 = da.from_array(nparr, chunks=(10, 1))
+darr3 = da.from_array(nparr, chunks=(10, 10))
```

Comment:
FWIW in NumPy's implementation 2-D transforms either proceed through the same code path as N-D transforms or they actually call the N-D transform. So they really are just a special case of the N-D transform.